### DISCUSSION TITLE
Effects and sound effects

### DISCUSSION THREAD

**Discussion by dodzzDev** [WorldBoxOpenMods/ModLoader#27] [Ideas]
I like this mod, but it could be improved in a few ways. First, when the cannons are fired at buildings or civilians, it would be much cooler if there were explosion effects along with sound, similar to throwing a grenade in the game. You can hear the sound and see the explosion, and I think adding that feature would enhance the experience.

**Comment by MelvinShwuaner** [comment]
I added it a long time ago.
